src/Webhooks.ts

```diff
@@ -218,9 +223,11 @@ function parseEventDetails(
   encodedHeader: WebhookHeader,
   expectedScheme: string
 ): WebhookParsedEvent {
-  const decodedPayload = Buffer.isBuffer(encodedPayload)
-    ? encodedPayload.toString('utf8')
-    : encodedPayload;
+  const textDecoder = new TextDecoder('utf8');
```

Comment:
nit: inconsistent usage of `textDecoder` as a variable vs `new TextDecoder`. Also inconsistent 'utf8' parameter usage.

Reply:
Ah good point, it defaults to 'utf8' but we should be explicit.